cirq/google/optimizers/convert_to_sycamore_gates.py

```diff
@@ -75,10 +75,20 @@ def _is_native_sycamore_op(self, op: ops.Operation) -> bool:
         Returns:
             True if the operation is native to the gmon, false otherwise.
         """
-        return (isinstance(op, ops.GateOperation) and isinstance(
-            cast(ops.GateOperation, op).gate,
-            (SycamoreGate, ops.MeasurementGate, ops.PhasedXPowGate,
-             ops.XPowGate, ops.YPowGate, ops.ZPowGate)))
+        gate = op.gate
+
+        if isinstance(
+                gate,
+            (SycamoreGate, ops.MeasurementGate, ops.PhasedXZGate,
```

Comment:
Maybe add a quick note that this also adds the PhasedXZGate?

Reply:
Will note in the commit message.